modules/ROOT/pages/partner-manager-identifiers.adoc

```diff
@@ -1,3 +1,90 @@
 = Identifiers
 
-Identifiers are used to identify the sender and receiver partners that are involved in a B2B transaction. For X12 messages, the identifiers in the ISA and GS headers provide valuable information that enables identification of the appropriate flow to use to process the transaction.
\ No newline at end of file
+Identifiers are used to identify the sender and receiver partners that are involved in a B2B transaction. For X12 messages, the identifiers in the ISA and GS headers provide valuable information that enables identification of the appropriate flow to use to process the transaction.
+
+== Add and Configure a New Identifier
+
+. In Partner Manager, select the name of your host organization in the left navigation menu. 
+. In the *Identifiers* section, click *New*. 
+. In the *New Identifier* dialog, select the identifier type from the drop-down list:
+* <<as2-identifier,*AS2 Identity*>>
+* <<reference-id,*Reference ID*>>
+* <<x12-gs,*X12 - GS*>>
+* <<x12-isa,*X12 - ISA*>>
+. In *Identifier Value*, enter a value to uniquely identify your identifier. 
```

Comment:
It's not a big deal, but there are a lot of "identify"s/ers in this sentence. Can we say something like "to uniquely name"?